extensions/install.php

```diff
@@ -1168,21 +1172,17 @@ private function preventUninstallRedcore($parent)
 		$manifest  = $this->getManifest($parent);
 		$isRedcore = 'COM_REDCORE' === (string) $manifest->name;
 
-		if ($isRedcore)
+		if ($isRedcore && method_exists('RComponentHelper', 'getRedcoreComponents'))
 		{
-			if ((method_exists('RComponentHelper', 'getRedcoreComponents')) && $components = RComponentHelper::getRedcoreComponents())
-			{
-				$app = JFactory::getApplication();
-
-				$message = sprintf(
-					"Cannot uninstall redCORE because the following components are using it: <br /> [%s]",
-					implode(",<br /> ", $components)
-				);
-
-				$app->enqueueMessage($message, 'error');
-
-				$app->redirect('index.php?option=com_installer&view=manage');
-			}
+			$components = RComponentHelper::getRedcoreComponents();
```

Comment:
this line should also be in the if statement because otherwise you will never be able to uninstall it. This line checks if there is actually some extension dependent on redCORE